### PR TITLE
Extensions

### PR DESCRIPTION
**Pull request by yeegor**
In order to set up the first extension:
```
# in root/vendor folder
mkdir yeegor && cd $_
git clone git@github.com:yeegor/scandipwa-paypal.git
```
Modify extensions.json as follows: 
```
{
    "extensions": {
        "PayPal": ["vendor/yeegor/src/plugin/PayPal.plugin.js"]	
    }	
}
```
Note: You need to recompile theme each time you change extensions.json

### CODE REVIEW

**Comment by yeegor** [comment]
We have class ProductGalleryBaseImageContainer that extends React.Component. Maybe we have some other classes like this one. These are not instance-extensible, but are class-extensible.

File: scandipwa-paypal/src/app/component/PayPal/PayPal.style.scss [hunk 1]
```diff
@@ -10,8 +10,11 @@
  */
 
 .PayPal {
-    margin-top: 2rem;
     min-height: 5rem;
+
```
Comment (Danpiel) [inline scandipwa-paypal/src/app/component/PayPal/PayPal.style.scss]: Codacy Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/manual/ScandiPWA/base-theme/pullRequest?prid=5162789)

**Comment by Danpiel** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/ScandiPWA/base-theme/pullRequest?prid=5162789&bid=16950923)

**Comment by alfredsgenkins** [comment]
Merged in #596

**Pull request closed without merging**